core/src/View/Kokkos_BasicView.hpp

```diff
@@ -96,6 +96,16 @@ transform_kokkos_slice_to_mdspan_slice(const T &s) {
   return KokkosSliceToMDSpanSliceImpl<T>::transform(s);
 }
 
+// FIXME_HPX spurious warnings like
+// error: 'SR.14123' may be used uninitialized [-Werror=maybe-uninitialized]
+#if defined(KOKKOS_ENABLE_HPX)
+#pragma GCC diagnostic push
+#if !defined(__clang__)
+#pragma GCC diagnostic ignored "-Wmaybe-uninitialized"
+#endif
+#pragma GCC diagnostic ignored "-Wuninitialized"
+#endif
+
```

Comment:
I managed to reproduce this with just the serial backend enabled over here: https://github.com/msimberg/kokkos/actions/runs/14673110353/job/41184658454#step:10:343 (note that on the log posted in the PR description the error is also triggered from a serial backend test). It's otherwise the same as the HPX configuration, so it's likely something from the fedora image that's triggering it compared to the other serial CI configuration (there are a few other differences, not sure which are relevant)?